package.json

```diff
@@ -68,6 +69,7 @@
     "minimatch": "3.0.4",
     "ms": "2.1.3",
     "nanoid": "3.1.20",
+    "node-sass": "^5.0.0",
```

Comment:
`node-sass` needs to be in `devDependencies`.  so run `npm rm node-sass && npm i node-sass -D` which will update `package-lock.json` as well (I think)

I don't see where the `sass` script is actually run, though. We won't see any changes on Netlify until it runs automatically.  I think what we should do is:

1. Revert the change on L49 of `package.json`
2. Apply the following diff to `package-scripts.js`:

```diff
diff --git a/package-scripts.js b/package-scripts.js
index 213be777..430e28e8 100644
--- a/package-scripts.js
+++ b/package-scripts.js
@@ -263,9 +263,14 @@ module.exports = {
     docs: {
       default: {
         script:
-          'nps docs.clean && nps docs.api && eleventy && nps docs.linkcheck && node scripts/netlify-headers.js docs/_site >> docs/_site/_headers',
+          'nps docs.clean && nps docs.scss && nps docs.api && eleventy && nps docs.linkcheck && node scripts/netlify-headers.js docs/_site >> docs/_site/_headers',
         description: 'Build documentation'
       },
+      scss: {
+        script: 'node-sass -w docs/scss/ -o docs/css/ --recursive',
+        description: 'Compile SCSS',
+        hiddenFromHelp: true
+      },
       production: {
         script: 'nps docs && nps docs.postbuild',
         description: 'Build docs for production'
```

We'll also need to ensure that the generated files are added to `.gitignore` if not already.